openml/config.py

```diff
@@ -88,6 +88,7 @@ def set_file_log_level(file_output_level: int):
     "cachedir": os.path.expanduser(os.path.join("~", ".openml", "cache")),
     "avoid_duplicate_runs": "True",
     "connection_n_retries": 2,
+    "max_retries": 20,
```

Comment:
Could you please increase the default number of retries to let's say 5 or so? There are still a few stochastic failures that can potentially be remedied by trying again.